src/go/wsl-helper/cmd/kubeconfig.go

```diff
@@ -42,80 +47,138 @@ var kubeconfigCmd = &cobra.Command{
 	Long:  `This command configures the Kubernetes configuration inside a WSL2 distribution.`,
 	Args:  cobra.ExactArgs(0),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		configPath := kubeconfigViper.GetString("kubeconfig")
+		winConfigPath := kubeconfigViper.GetString("kubeconfig")
 		enable := kubeconfigViper.GetBool("enable")
-		show := kubeconfigViper.GetBool("show")
 
-		if configPath == "" {
+		if !enable {
+			return nil
+		}
+
+		if winConfigPath == "" {
 			return errors.New("Windows kubeconfig not supplied")
 		}
 
-		_, err := os.Stat(configPath)
+		_, err := os.Stat(winConfigPath)
```

Comment:
The stat is unnecessary; `readKubeConfig` five lines down will return an error if anything is wrong.

We had it before because we just did a symlink and never read the file.